src/Blueprint/Blueprint/Cmdlets/RemoveAzureRMBlueprintAssignment.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.Azure.Commands.Blueprint.Models;
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.PowerShell.Cmdlets.Blueprint.Properties;
+using Microsoft.Azure.Commands.Blueprint.Common;
+using ParameterSetNames = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants.ParameterSetNames;
+using ParameterHelpMessages = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants.ParameterHelpMessages;
+using BlueprintConstants = Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants;
+
+
+namespace Microsoft.Azure.Commands.Blueprint.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BlueprintAssignment", SupportsShouldProcess = true, DefaultParameterSetName = ParameterSetNames.DeleteBlueprintAssignmentByName), OutputType(typeof(PSBlueprintAssignment))]
+    public class RemoveAzureRmBlueprintAssignment : BlueprintCmdletBase
+    {
+        #region Parameters
+        [Parameter(ParameterSetName = ParameterSetNames.DeleteBlueprintAssignmentByObject, Position = 0, Mandatory = true, ValueFromPipeline = true, HelpMessage = ParameterHelpMessages.AssignmentSubscriptionId)]
+        [Parameter(ParameterSetName = ParameterSetNames.DeleteBlueprintAssignmentByName, Position = 0, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.AssignmentSubscriptionId)]
+        [ValidateNotNullOrEmpty]
+        public string SubscriptionId { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.DeleteBlueprintAssignmentByName, Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.BlueprintAssignmentName)]
+        [ValidateNotNull]
+        public string Name { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.DeleteBlueprintAssignmentByObject, Position = 1, Mandatory = true, ValueFromPipeline = true, HelpMessage = ParameterHelpMessages.BlueprintAssignmentObject)]
+        public PSBlueprintAssignment Assignment { get; set; }
```

Comment:
@filizt this should be renamed to `-InputObject` (you can add an alias to `-Assignment`)

Reply:
Renamed.

---
In reply to: [261403238](https://github.com/Azure/azure-powershell/pull/8537#discussion_r261403238) [](ancestors = 261403238)